dts/arm/st/l4/stm32l452.dtsi

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2018 Georgij Cernysiov
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/l4/stm32l4.dtsi>
```

Comment:
You should be able to include another l4 .dtsi such as stm32l433 for instance

Reply:
I see, now it shall be the right way